sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Release History
 
+## 5.0.0-preview.3 (2019-09-09)
+
+- Added synchronous `EventHubConsumer` and `EventHubProducer`.
+- Added support for balancing partitions across multiple instances of `EventProcessor`.
+- Added `EventProcessorBuilder` to create `EventProcessor` and removed that functionality from `EventHubClientBuilder`.
+- Removed `CheckpointManager`.
```

Comment:
But why??

As a customer reading the changelog to decide whether or not I should update, I'd like to know why the `CheckpointManager` was removed, from where and how does that affect me.